### PR TITLE
Offline fix on linux

### PR DESCRIPTION
**Pull request by liealex**

### CODE REVIEW

File: sdk/src/connections/offline/offline_depth_sensor.cpp [hunk 1]
```diff
@@ -1,7 +1,7 @@
 #include "offline_depth_sensor.h"
 #include "tofi/tofi_config.h"
 #include <cstring>
-#ifdef UNIX
+#ifdef __linux__
```
Comment (dNechita) [inline sdk/src/connections/offline/offline_depth_sensor.cpp:4]: The commit addressing these changes says "Solved offline problem on linux".
So, what was the problem? Can you describe it?

Comment (liealex) [reply sdk/src/connections/offline/offline_depth_sensor.cpp:4]: "UNIX" was the wrong identifier for linux. I've changed it to "__linux__" so the code will run correctly when running on linux.

File: examples/tof-viewer/src/ADIMainWindow.cpp [hunk 2]
```diff
@@ -404,7 +404,10 @@ void ADIMainWindow::render() {
         ImGui_ImplOpenGL3_NewFrame();
         ImGui_ImplGlfw_NewFrame();
         ImGui::NewFrame();
-        handleInterruptCallback();
+        if (!m_callbackInitialized) {
+            handleInterruptCallback();
+            m_callbackInitialized = true;
+        }
```
Comment (dNechita) [inline examples/tof-viewer/src/ADIMainWindow.cpp:410]: Why the need for these changes? The commit message says "Added offline condition in handleInterruptCallback". I can't find any offline condition within these changes.

Comment (liealex) [reply examples/tof-viewer/src/ADIMainWindow.cpp:410]: Indeed there is no offline changes in this commit, I will make a separate pr for these changes.

Comment (liealex) [reply examples/tof-viewer/src/ADIMainWindow.cpp]: Changes removed

**Review by dNechita — APPROVED** [review]

**Comment by liealex** [comment]
> Are there other occurrences of UNIX within the codebase that need fixing?

No.

**Pull request merged**